tests/unittests/media/client/main/mediaPipeline/SetVolumeTest.cpp

```diff
@@ -44,17 +46,17 @@ class RialtoClientMediaPipelineSetVolumeTest : public MediaPipelineTestBase
  */
 TEST_F(RialtoClientMediaPipelineSetVolumeTest, setVolumeSuccess)
 {
-    EXPECT_CALL(*m_mediaPipelineIpcMock, setVolume(m_kVolume)).WillOnce(Return(true));
```

Comment:
Add a test to test the API when only volume is passed it so easetype and duration should be default.